### PR TITLE
chore: update translations

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Adds missing translations

Translations came from translators

### CODE REVIEW

File: i18n/data/es.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
   "UIKit.LocalizedTextInput.collapse": "Ocultar lenguajes ({remainingLanguages})",
   "UIKit.LocalizedTextInput.expand": "Mostrar todos los lenguajes ({remainingLanguages})",
   "UIKit.LocalizedTextInput.missingRequiredField": "Este campo es obligatorio. Proporcione un valor al menos.",
-  "UIKit.MoneyField.highPrecision": "",
-  "UIKit.MoneyInput.highPrecision": "",
+  "UIKit.MoneyField.highPrecision": "Precio de alta precisión",
```
Comment (montezume) [inline i18n/data/es.json]: thanks to @qmateub

File: i18n/data/zh-CN.json [hunk 2]
```diff
@@ -6,11 +6,11 @@
   "UIKit.AsyncSelectInput.noOptionsMessageWithInputValue": "无选项",
   "UIKit.AsyncSelectInput.noOptionsMessageWithoutInputValue": "无选项",
   "UIKit.AsyncSelectInput.placeholder": "选择...",
-  "UIKit.CalendarHeader.nextMonthTooltip": "",
-  "UIKit.CalendarHeader.nextYearTooltip": "",
-  "UIKit.CalendarHeader.previousMonthTooltip": "",
-  "UIKit.CalendarHeader.previousYearTooltip": "",
-  "UIKit.CalendarHeader.todayTooltip": "",
+  "UIKit.CalendarHeader.nextMonthTooltip": "下个月",
```
Comment (montezume) [inline i18n/data/zh-CN.json]: 哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈

**Comment by tdeekens** [comment]
Has this been approved by translators?

**Comment by montezume** [comment]
@tdeekens yes the calendar header ones came from the translators...,

**Pull request merged**